src/components/anti-everyone.ts

```diff
@@ -6,6 +6,11 @@ import { M } from "../utils/debugging-and-logging.js";
 
 const failed_everyone_re = /(?:@everyone|@here)/g; // todo: word boundaries?
 
+/**
+ * @TODO: This will likely grow rather large, thus it may be a good idea to offload this into a full database/disk storage
+ */
+const replies = new Map();
```

Comment:
A `SelfClearingMap<string, Discord.Message[]>` would be best here

Reply:
Sounds good, I didn't have enough knowlege about Wheatly's internals, so this sounds a lot better, and resolves my TODO